packages/list/src/__tests__/list-command.spec.ts

```diff
@@ -12,14 +12,16 @@ vi.mock('@lerna-lite/core', async () => ({
   collectUpdates: (await vi.importActual<any>('../../../core/src/__mocks__/collect-updates')).collectUpdates,
   PackageGraph: (await vi.importActual<any>('../../../core/src/package-graph/package-graph')).PackageGraph,
   getPackages: (await vi.importActual<any>('../../../core/src/project/project')).getPackages,
+  getFilteredPackages: (await vi.importActual<any>('../../../core/src/filter-packages')).getFilteredPackages,
 }));
 
 vi.mock('@lerna-lite/list', async () => await vi.importActual<any>('../list-command'));
-vi.mock('@lerna-lite/filter-packages', async () => await vi.importActual<any>('../../../filter-packages/src/get-filtered-packages'));
 
 // mocked modules
 import { collectUpdates, ListCommandOption, logOutput } from '@lerna-lite/core';
 
+vi.mock('../../../core/src/utils/collect-updates/collect-updates.js', async () => await vi.importActual('../../../core/src/__mocks__/collect-updates'));
```

Comment:
could you move this mock with the other mocks above? around line 17 would be better